src/component/main/index.tsx

```diff
@@ -42,15 +43,15 @@ function Main(props: MainProps) {
   const hasVentures =
     ventureContext && ventureContext?.ventures?.length > 0 ? true : false;
 
-  const {
-    handleSubmit,
-    register,
-    reset,
-    watch,
-    control,
-    formState: { errors },
-  } = useForm();
-  const watchData = watch();
+  const [ventureData, setVentureData] = useState<VentureFormData>({
```

Comment:
Hoisting state to `main` using `watch` in the child components and `useState` in the parent instead of keeping all of the form state in this component.